paper/paper.md

```diff
@@ -0,0 +1,432 @@
+---
+title: 'Teaching Python for Data Science: Collaborative development of a modular & interactive curriculum'
+tags:
+  - Data Science
+  - Python
+  - Jupyter
+  - Girls Who Code
+authors:
+  - name: Marlena Duda
+    orcid: TODO
+    affiliation: 1
+  - name: Kelly L. Sovacool
+    orcid: 0000-0003-3283-829X
+    affiliation: 1
+  - name: Negar Farzaneh
+    orcid: TODO
+    affiliation: 1
+  - name: Vy Nguyen
+    orcid: TODO
+    affiliation: 1
+  - name: Sarah Haynes
+    orcid: 0000-0003-3225-1691
+    affiliation: 6
+  - name: Hayley Falk
+    orcid: 0000-0002-5198-1836
+    affiliation: 1
+  - name: Katherine L. Furman
+    orcid: 0000-0002-0340-4292
+    affiliation: "3, 4"
+  - name: Logan A. Walker
+    orcid: 0000-0002-5378-3315
+    affiliation: "5, 1"
+  - name: Rucheng Diao
+    orcid: TODO
+    affiliation: 1
+  - name: Morgan Oneka
+    orcid: TODO
+    affiliation: 1
+  - name: Audrey Drotos
+    orcid: TODO
+    affiliation: TODO
+  - name: Alana Woloshin
+    orcid: 0000-0002-5224-4221
+    affiliation: 7
+  - name: Gabrielle Dotson
+    orcid: TODO
+    affiliation: 1
+  - name: April Kriebel
+    orcid: 0000-0003-0008-9044
+    affiliation: 1
+  - name: Lucy Meng
+    orcid: TODO
+    affiliation: TODO
+  - name: Stephanie N. Thiede
+    orcid: 0000-0003-0173-4324
+    affiliation: 2
+  - name: Shweta Ramdas
+    orcid: TODO
+    affiliation: TODO
+  - name: Zena Lapp^[corresponding author]
+    orcid: 0000-0003-4674-2176
+    affiliation: 1
+  - name: Brooke N. Wolford^[corresponding author]
+    orcid: 0000-0003-3153-1552
+    affiliation: 1
+affiliations:
```

Comment:
Could you add my affiliations from EHS? Department of Environmental Health Sciences, University of Michigan